### ISSUE TITLE
 the CTC activation function

### ISSUE THREAD

**Issue by WillQuCD** [SeanNaren/deepspeech.pytorch#271]
Thank you for your contribution. 

"Speech Recognition using DeepSpeech2 and the CTC activation function. Edit"
But the CTC in Deep Speech is loss function and not activation function.

**Comment by SeanNaren** [comment]
Fixed! Thanks :)
